### ISSUE TITLE
Geodata Fiberkabler slettes automatisk

### ISSUE THREAD

**Issue by ClockEnd** [JosefAssad/SeMaWi#192]
http://10.192.224.40:12345/index.php?title=Speciel:Loglister&page=Geodata+Fiberkabler kan se den bliver slettet/ikke overført - hvad var status? Noget som MapCentia skulle se på

**Comment by JosefAssad** [comment]
Hm. Så vidt jeg husker så handlede det i bund og grund om bevaring af GUID for geodata tabeller. Tesen i øjeblikket er, at pågældende tabel fik ændret GUID på GC2 siden. Når det sker så slettes siden med det gamle GUID i SeMaWi og der oprettes ny side med det nye GUID. Men den tilknyttede dokumentationsside peger stadig på den gamle GUID.

Så hvis det er korrekt, så er spørgsmålet hvorfor og hvordan GUID'et er ændret i GC2 for samme tabel. Jeg mener @baffioso havde nogle ideer til, brugsmønstre som kan føre til ændret GUID?

I wiki'en er det nemt nok at manuelt håndtere de sjældne tilfælde hvor en geodata side får ny GUID, vi kan nemlig finde dukomentationssiden frem og omdøbeden, hvilket efterlader en omdirigering så siden alligevel fungerer selv hvis der skiftes tilbage til det gamle GUID. Kan du fortælle hvad geodata tabellen hed?

**Comment by ClockEnd** [comment]
Geodata Fiberkabler
GUID: ceba781d-be74-4a7f-b160-6cd09376502a

**Comment by JosefAssad** [comment]
Yep, jeg fandt dokumentationssiden med et gammelt GUID og har "movet" siden så den nu fungerer som den gjorde før.

Det tyder mere og mere som at tabellen i GC2 fik ny GUID hvilket gjorde at den "tabte" den tilknyttede dokumentationsside. Så mangler der bare at finde ud af, hvordan tabellen fik nyt GUID i GC2. Måske har @baffioso teorier her?

**Comment by baffioso** [comment]
Beklager det sene svar. Jeg har ikke umiddelbart teorier, da det ikke lader til at tabellen har været slettet og tilføjet igen. Har lige været inde og tjekke GUID og kan se den ikke har ændret sig siden @ClockEnd sidste kommentar

**Comment by JosefAssad** [comment]
Alright. Vi lader den ligge som "open", men i backlog. Når (hvis?) det sker igen så ser vi på, om ikke vi kan snuse os frem til hvor UUID'en bliver ændret.

Nu ved vi i hvert fald at den workaround med at oprette en redirect fra dokumentationssiden med den gamle UUID til den nye fungerer fint nok, så det er rigtig godt. :)
